delivery_roulier_accounts_on_sales_shop/__openerp__.py

```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
+# @author Raphael Reverdy <raphael.reverdy@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Delivery accounts on Shop',
+    'version': '9.0.0.0.0',
+    'author': 'Akretion',
+    'summary': 'Set an account for delivery on shop',
+    'maintainer': 'Akretion, Odoo Community Association (OCA)',
+    'category': 'Warehouse',
+    'depends': [
+        'keychain',
+        'delivery_roulier',
+        'sale_shop',
```

Comment:
Its suitable to have sale_shop (and all modules in dependency) in this repository (ak-odoo-incubator) or other public repository.